File: Homework 2/turker_ege.py
t = 0
while (t > 0):
    print ("Pretest")
    
emptyList = []
for val in emptyList:
    print (val)
    
i = 0
while (i < 3):
    print("For Loop: ", i) 
    j = 0
    while (j < 10):
        print("While Loop: ", j)
        j = j + 1
        if ( j == 2):
            break
    if ( i != 2):
        i = i + 1
        continue
    i = i + 1
    print("End of Outer Loop")
    

File: Homework 2/turker_ege.dart
void main() {
  
  loopFor:
  for(var t = 0; t > 10; t++){
    print("Pretest");
  }
  
  loopWhile:
  var t = 0;
  while(t > 10){
    print("Pretest");
  }
  
  t = 0;
  loopDoWhile:
  do{
    print("Posttest");

  }while (t > 10);

  loopCont:
  for (var i = 0 ; i < 3; i++){
    print("For Loop: ${i}");
    var j = 0;
    loopBreak:
    while(j < 10){
      print("While Loop: ${j}");
      j++;
      if(j == 2) break loopBreak;
    }
    if(i != 2) continue loopCont;
    print("End of For Loop");
  }
}